delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -1041,21 +1040,32 @@ def updateServiceInDB(service_id_in_db, devApiID, region, statusval) {
 def loadEventsModule() {
 }
 
+	
 def loadServiceInfo() {
 	def token = setCredentials()
 	def url = "$g_base_url/platform/services?domain=$domain&service=$service&isAdmin=true"
 	def curlOutput = sh (script: "curl -H \"Content-Type: application/json\" \
-			-H \"Authorization: $token\" \"$url\" | $jq -r '.data[0] | [.created_by] | join(\",\")'", returnStdout: true)
-	def values = curlOutput.split(',')
-
+			-H \"Authorization: $token\" \"$url\"", returnStdout: true)
+  	echo "CURL o/p: $curlOutput"
+  	
+	def jsonParser = new groovy.json.JsonSlurper()
```

Comment:
Can we initialize this (JsonSlurper) globally for this file, instead of multiple times in different functions?

Reply:
Tried to initialize  globally but it gave error as ' java.io.NotSerializableException' caused by JsonSlurper.  So initializing needs to be done in a method.